### ISSUE TITLE
How to create a custom boot2k8s iso image?? I want to create a VM based on my custom image.

### ISSUE THREAD

**Issue by sharminakteranee** [skippbox/boot2k8s#8]

**Comment by sebgoa** [comment]
the Makefile calls a docker build. So if you want to tweak the image, you need to edit the Dockerfile and the scripts that are being put in the docker image.

**Comment by sharminakteranee** [comment]
I tried that, but there is the make_iso.sh script is not provided that is stated on Dockerfile

**Comment by sebgoa** [comment]
Can you give some logs.
make_iso.sh is part of the boot2docker/boot2docker image. I just downloaded it and the script it there:

```
$ docker run -ti boot2docker/boot2docker ls -l make_iso.sh
-rwxr-xr-x 1 root root 2021 Mar 10 22:08 make_iso.sh
```

Since the boot2k8s Dockerfile starts with a FROM boot2docker/boot2docker , you should get the script, unless you changed the base image.

**Comment by sharminakteranee** [comment]
Yes, I have tried using another base image, So the script is not available

**Comment by sebgoa** [comment]
if you change the base image in our Dockerfile, than yes you will not have access to make_iso.sh
